### PR TITLE
Documented Use Extension to Commit page

### PR DESCRIPTION
**Pull request by bprerak**
I have documented Use Extension to Commit page about jovian jupyter extention

### CODE REVIEW

File: docs/nbext/commit.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+When running `jovian.commit` for first time you'll be prompted to provide an API key, which you can get from your [Jovian](https://jvn.io) (or Jovian Pro) account. By clicking on `API key` button, key will be copied to the clipboard.
+
+<img src="https://i.imgur.com/taLLUVd.png" class="screenshot">
+
+#### Here's what `jovian.commit` does:
+
+It saves the current state of the notebook. It captures and uploads the required dependencies from the python environment and python environment itself. Capturing the python environment and dependencies ensures that the notebook can be reproduced and executed easily. Once the process has been completed a link will be displayed in a pop up, which you can use to view and share your notebook with friends or colleagues.
+
+
+<img src="https://i.imgur.com/1cFeiC7.gif" class="screenshot">
```
Comment (PrajwalPrashanth) [inline docs/nbext/commit.md]: Hey @bprerak these lines of documentation suggests about `jovian.commit`. Use the Jupyter extension(Toolbar button which says `/j commit`) and document the steps involved in it.

File: README.md [hunk 2]
```diff
@@ -112,3 +112,4 @@ To run the tests, run the following command in the project root:
 - Siddhant Ujjain
 - Mayank Saboo
 - Zilong Ye
+- Prerakbhai Bhakta
```
[the remark this answers is not in the document]
Comment (bprerak) [reply README.md]: How can I address this issue?

File: docs/nbext/commit.md (hunk 1 above)
Comment (bprerak) [reply docs/nbext/commit.md]: I have addressed the issues, can you please review it. 
Thank You

File: docs/nbext/commit.md [hunk 3]
```diff
@@ -1 +1,35 @@
...
+$ jovian disable-ext
+```
+
+The changes are observed when the webpage of the notebook is refreshed.
+
+
+#### Here's what `/j Commit` button does:
+
+It saves the current state of the notebook. It captures and uploads the required dependencies from the python environment and python environment itself. Capturing the python environment and dependencies ensures that the notebook can be reproduced and executed easily. Once the process has been completed a link will be displayed in a pop up, which you can use to view and share your notebook with friends or colleagues.
+
```
Comment (PrajwalPrashanth) [inline docs/nbext/commit.md]: Purge the credentials and try the extension to explain all the states involved.
* API Key
* Invalid API Key
* Successful commit

Comment (PrajwalPrashanth) [reply docs/nbext/commit.md]: Upload the screenshots to imgur and link them too as done in the other sections of the docs site.

**Pull request closed without merging**